src/jit/target.h

```diff
@@ -1440,7 +1440,7 @@ typedef unsigned short regPairNoSmall; // arm: need 12 bits
 
   // The registers trashed by the CORINFO_HELP_STOP_FOR_GC helper
   // See vm\arm\amshelpers.asm for more details.
-  #define RBM_STOP_FOR_GC_TRASH     (RBM_CALLEE_TRASH & ~(RBM_FLOATRET | RBM_INTRET))
+  #define RBM_STOP_FOR_GC_TRASH     (RBM_CALLEE_TRASH & ~(RBM_DOUBLERET | RBM_INTRET))
```

Comment:
> I'm not sure whether it's best to do that, or to avoid having to push and pop them all in JIT_RareDisableHelper. But it seems that they should be consistent.

Right, we should keep consistency.
I will take a look again.

BTW I think this decision may affect overall code quality, i.e. size of code (caller vs. callee),  performance (caller-saves vs. callee-saves) and etc.